src/main/controllers/CreateMediaAccountController.ts

```diff
@@ -1,23 +1,46 @@
-import { PipRequest } from '../models/request/PipRequest';
-import { Response } from 'express';
-import { CreateAccountService } from '../service/createAccountService';
-import { cloneDeep } from 'lodash';
+import {PipRequest} from '../models/request/PipRequest';
+import {Response} from 'express';
+import {CreateAccountService} from '../service/createAccountService';
+import {cloneDeep} from 'lodash';
+import {ManualUploadService} from '../service/manualUploadService';
 
 const createAccountService = new CreateAccountService();
+const manualUploadService = new ManualUploadService();
 
 export default class CreateMediaAccountController {
   public get(req: PipRequest, res: Response): void {
-    res.render('create-media-account', req.i18n.getDataByLanguage(req.lng)['create-media-account']);
+    const formCookie = req.cookies['formCookie'];
```

Comment:
why do you need cookie? On manual upload we needed it to pass data between pages, this ticket seems to stick to one page at all times?

Reply:
we need it only because on the re-render if errors the data are pre-filled in the form

Reply:
You don't need this for setting values into input fields. Please read the documentation. 
![Capture](https://user-images.githubusercontent.com/25640912/156167516-4cff5740-c6a9-4d37-8998-909d4c14a4dd.PNG)
![set](https://user-images.githubusercontent.com/25640912/156167518-d8463731-6ca1-4e1c-8dce-f430ec9988aa.PNG)